### PR TITLE
:bug: Isolate widget css from guest page css

### PR DESCRIPTION
**Pull request by lboyette-okta**
@rchild-okta 
@haishengwu-okta

### CODE REVIEW

**Comment by lboyette-okta** [comment]
I haven't yet tested the affect on okta-core, but @alanzhou-okta opened this PR quite a while ago: https://github.com/okta/okta-core/pull/17361

File: assets/sass/okta-theme.scss [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
 
 // scss-lint:disable IdSelector
-#okta-sign-in {
+#okta-sign-in.auth-container {
```
Comment (lboyette-okta) [inline assets/sass/okta-theme.scss]: I made the theme more specific. The argument is that themes should be able to override the base at any point.

File: assets/sass/_base.scss [hunk 2]
```diff
@@ -1,20 +1,3 @@
...
-  line-height: 1.4;
-  background-color: $secondary-bg-color;
-  min-width: 100%;
-  min-height: 100%;
-  color: $medium-text-color;
-  position: relative;
-  overflow: auto;
-}
-
 a {
```
Comment (rchild-okta) [inline assets/sass/_base.scss]: These should also be specific to the widget right (the remaining classes in this file)?

File: assets/sass/_container.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// scss-lint:disable IdSelector
+#okta-sign-in {
+// scss-lint:enable IdSelector
+
+  // Html and body styles in _base.scss that were moved here
+  font-family: $fonts;
+  font-size: $font-size;
+  font-weight: $font-weight-body;
+  line-height: 1.4;
+  background-color: $secondary-bg-color;
```
Comment (rchild-okta) [inline assets/sass/_container.scss]: Hmm, so some of these don't make sense here (and in general, will just need to be ported to okta-core), i.e:
- Is background-color here for the container, or for the background page?
- min-height - we're setting the height below (and don't really want the widget container to be 100%)
- position: relative and overflow: auto were fixes on the container for embedded scenarios - probably don't belong here, and will definitely need to be tested in okta-core when we port these over

File: assets/sass/_base.scss (hunk 2 above)
Comment (lboyette-okta) [inline assets/sass/_base.scss]: I agree

File: assets/sass/_container.scss (hunk 3 above)
Comment (lboyette-okta) [inline assets/sass/_container.scss]: - I think originally it came from the page background. `background-color` can be removed
- `min-height` can be removed. What's odd is that this renders normally.
- `position` and `overflow` can be removed

File: assets/sass/_base.scss (hunk 2 above)
Comment (lboyette-okta) [inline assets/sass/_base.scss]: yeah, I guess it doesn't make sense to create _container.scss at all. Everything can just be namespaced in _base.scss

Comment (lboyette-okta) [inline assets/sass/_base.scss]: wait, they are namespaced - they're imported within an `#okta-sign-in` tag in okta-sign-in.scss

File: assets/sass/_container.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    width: 100%;
+  }
+
+  @include max-height-mq(750px) {
+    margin-top: 0;
+  }
+}
+
+// scss-lint:disable IdSelector
+#okta-sign-in.no-beacon .auth-content {
```
Comment (haishengwu-okta) [inline assets/sass/_container.scss]: nested into okta-sign-in namespace?

Comment (lboyette-okta) [inline assets/sass/_container.scss]: the `.no-beacon` makes it so I can't. However, I can combine the `.auth-content` class into the same namespace with the `.auth-header` below

Comment (haishengwu-okta) [inline assets/sass/_container.scss]: I thought we could nest it like `&.no-beacon`

```
#foo {
  &.bar .baz {
    color: red;
  }
}
```
 
generates

```
#foo.bar .baz {
  color: red;
}
```

**Pull request closed without merging**